ddtrace/contrib/requests/patch.py

```diff
@@ -31,7 +31,6 @@ def patch():
     _w("requests", "Session.send", _wrap_send)
     Pin(_config=config.requests).onto(requests.Session)
 
-
```

Comment:
unrelated change?

Reply:
Yup, it was added while fixing merge conflicts. I'll reset this branch